### ISSUE TITLE
__concurix_tracer is not defined

### ISSUE THREAD

**Issue by josip** [strongloop/strong-supervisor#110]
We had 1.5.1 installed on a machine which would at random points throw this error:

```
2015-04-20T07:36:20.918Z pid:18804 worker:1 ERR [ReferenceError: __concurix_tracer is not defined] ReferenceError: __concurix_tracer is not defined
2015-04-20T07:36:20.919Z pid:18804 worker:1     at notes/all.appscript.vm:2:42
2015-04-20T07:36:20.919Z pid:18804 worker:1     at notes/all.appscript.vm:44:15
2015-04-20T07:36:20.919Z pid:18804 worker:1     at Script.(anonymous function) [as runInNewContext] (vm.js:41:22)
2015-04-20T07:36:20.919Z pid:18804 worker:1     at [redacted]
2015-04-20T07:36:20.919Z pid:18804 worker:1     at [redacted]/node_modules/async/lib/async.js:707:17
2015-04-20T07:36:20.919Z pid:18804 worker:1     at [redacted]/node_modules/async/lib/async.js:692:17
2015-04-20T07:36:20.919Z pid:18804 worker:1     at Array.forEach (native)
2015-04-20T07:36:20.919Z pid:18804 worker:1     at _each ([redacted]/node_modules/async/lib/async.js:59:42)
2015-04-20T07:36:20.919Z pid:18804 worker:1     at Object.taskComplete ([redacted]/node_modules/async/lib/async.js:690:13)
2015-04-20T07:36:20.919Z pid:18804 worker:1     at processImmediate [as _immediateCallback] (timers.js:345:15)
```

After a bit of googling it turned out that this "concurix" thing is used by strong-supervisor (or is planned to be used). I can provide you with exact where things would break if needed.

Downgrading to 1.4.1 "fixed" the problem.

**Comment by Setogit** [comment]
check strong-trace-instrument/function_trace.js and  strong-trace-core-tracer/tracer.js

**Comment by sam-github** [comment]
We are looking into this, thanks for the report.

**Comment by Setogit** [comment]
this is most likely fixed in strong-supervisor 1.5.2 .

**Comment by sam-github** [comment]
Can you reference the specific commit in 1.5.2 which fixed this?

**Comment by Setogit** [comment]
https://github.com/strongloop/strong-supervisor/commit/17941c30b7abf127a87d0a70047ff55fe4bfb72c

**Comment by sam-github** [comment]
@josip can you confirm this is fixed in 1.5.2?
